Documentation/Configuration/Typo3ConfVars/SYS.rst

```diff
@@ -433,6 +433,10 @@ configurations.
     the server. If :php:`*` all proxies defined in
     :ref:`[SYS][reverseProxyIP]<typo3ConfVars_sys_reverseProxyIP>` use SSL.
 
+    ..  versionadded:: 13.4
```

Comment:
should we also mention this in the news reverse proxy chapters or is it not that important? Any idea what this means to the end users?

Reply:
I have no clue what you mean with: `news reverse proxy chapters`